presto-main/src/test/java/com/facebook/presto/cost/TestCostCalculator.java

```diff
@@ -136,16 +138,16 @@ public void testRepartitionedJoin()
                 costs,
                 stats)
                 .cpu(12000 + 6000 + 1000 + 6000 + 1000)
-                .memory(1000)
-                .network(0);
+                .memoryUnknown()
+                .networkUnknown();
 
         assertCostEstimatedExchanges(
                 join,
                 costs,
                 stats)
                 .cpu(12000 + 6000 + 1000 + 6000 + 1000 + 6000 + 1000)
-                .memory(1000)
-                .network(6000 + 1000);
+                .memoryUnknown()
```

Comment:
It changes because underneath we get sth like '8 + NaN' in a place where previously it was (incorrectly) '8 + 0'. It has to stay this way until we have better (in terms of completeness) cost estimation for the involved cases.

I'd rather not compute estimates from partial knowledge, as the optimizer will end up preferring plans with fewer stats (and thus smaller cost).